account-kit/react/src/hooks/useSignMessage.ts

```diff
@@ -23,7 +21,7 @@ export type UseSignMessagedMutationArgs = BaseHookMutationArgs<
 >;
 
 export type UseSignMessageArgs = {
-  client: UseSmartAccountClientResult["client"] | undefined;
+  client: { account: { address: Hex } } | undefined;
```

Comment:
loosened this up a little since we aren't actually using it except to get the addr, but lmk if i should just leave as-is

Reply:
if i keep this change though, i sh ould prob at least change `address` to `Address` instead of `Hex`